src/platform/test/functional/services/dashboard/add_panel.ts

```diff
@@ -14,62 +14,68 @@ export class DashboardAddPanelService extends FtrService {
   private readonly retry = this.ctx.getService('retry');
   private readonly testSubjects = this.ctx.getService('testSubjects');
   private readonly flyout = this.ctx.getService('flyout');
-  private readonly common = this.ctx.getPageObject('common');
   private readonly header = this.ctx.getPageObject('header');
   private readonly savedObjectsFinder = this.ctx.getService('savedObjectsFinder');
-  private readonly browser = this.ctx.getService('browser');
   private readonly toasts = this.ctx.getService('toasts');
 
-  async clickOpenAddPanel() {
-    this.log.debug('DashboardAddPanel.clickOpenAddPanel');
+  async clickTopNavAddMenu() {
+    this.log.debug('DashboardAddPanel.clickTopNavAddMenu');
+    await this.testSubjects.click('dashboardAddTopNavButton');
+  }
+
+  async clickAddFromLibrary() {
```

Comment:
Thank you for renaming `clickOpenAddPanel` to `clickAddFromLibrary`. This naming has confused me so many times.